README.md

```diff
@@ -71,12 +71,22 @@ $ cd sbomnix
 $ nix-shell
 ```
 
-From the development shell, run `sbomnix` as follows:
+Keep in mind this doesn't add the various entrypoint binaries to your `PATH`
+directly. They are produced during the setuptools build.
+
+While you're in the devshell, you can run run `sbomnix` via the entrypoint file
+directly:
+
 ```bash
-$ sbomnix --help
+$ sbomnix/main.py --help
 ```
```

Comment:
I think we need to overview the documentation across the repo, and make sure it instructs correct usage.
We (I) can also do that in the follow-up PR later.